src/serviceconnector-passwordless/azext_serviceconnector_passwordless/_credential_free.py

```diff
@@ -635,24 +620,23 @@ def create_aad_user(self):
         except AzureConnectionError as e:
             logger.warning(e)
             # allow local access
-            from requests import get
-            ip_address = self.ip or get(IP_ADDRESS_CHECKER).text
-            self.set_target_firewall(True, ip_name, ip_address, ip_address)
+            ip_address = self.ip or get_local_ip()
+            if not ip_address:
+                self.set_target_firewall(
```

Comment:
make sure users are aware

Reply:
Yes, the empty ip_address is provided by user